cmd/openshift-install/destroy.go

```diff
@@ -49,6 +50,16 @@ func runDestroyCmd(cmd *cobra.Command, args []string) error {
 		return errors.Wrap(err, "Failed to destroy cluster")
 
 	}
+
+	store, err := asset.NewStore(rootOpts.dir)
```

Comment:
Can we create this store first and pass it in to `destroy.New`.  Then we can get the metadata JSON from the store, allowing us to drop [this][1] and DRY things up a bit.  I'm also fine punting that to follow-up work.

[1]: https://github.com/openshift/installer/blob/58486f9eea77136f89796691cb0823ee22201824/pkg/asset/cluster/cluster.go#L159-L171

Reply:
I didn't want to change the destroy code in this PR. we can follow up in another one and unify.